### ISSUE TITLE
OnResume() not work after Home button pressed and back to app

### ISSUE THREAD

**Issue by tkxel-android** [harism/android-pagecurl#4]
Hi, I have found a issue. 
In the app when I pressed Home button , and the again click on the app. It show me only back pages for curl, no images found. 
The same issue is when screen is screen. Then I just unlock the screen and it again show only black pages.

Same is when I try to put this in the "Tabed Activity". 1 tab is for curl. But when I switch to other tab , and then back to the curl page tab. It show only the black pages. No images to curl.

Please check this out.

Thanks

**Comment by harism** [comment]
I will take a look on this later today and have a guess what's causing the problem. Anyway, thanks for the feedback already now.

**Comment by harism** [comment]
I wasn't able to reproduce same situation on emulator but my guess is as follows (I committed fix proposal).
1. CurlRenderer receives a notification on onSurfaceCreated , meaning a new GL context has been created.
2. Lifeline of java classes differs from this - and in this particular case I had texture id stored in CurlMesh. Which is not valid after new GL context has been recreated but a new one has to be allocated.

As a fix I added a callback method from CurlRenderer.onSurfaceCreated for reallocating new texture ids and what ever might have to be reset due to new GL context creation.
